pmd-core/src/main/java/net/sourceforge/pmd/cli/internal/CliMessages.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cli.internal;
+
+/**
+ * @author Clément Fournier
+ */
+public final class CliMessages {
+
+    private CliMessages() {
+        // utility class
+    }
+
+    public static String errorDetectedMessage(int errors, String program) {
+        String anError = errors == 1 ? "An error" : errors + " errors";
+        return anError + " occurred while executing " + program + ".\n"
+            + "Run with --debug to see a stack-trace.\n"
```

Comment:
I think, CPD doesn't have a `--debug` flag... at least we don't have it documented: https://pmd.github.io/latest/pmd_userdocs_cpd.html
I'll create a new issue, this should be sorted at some point...

Reply:
See #3796